README.md

```diff
@@ -38,6 +38,93 @@ deactivate
 ```
 
 
+# Configuration
+
+
+## Main configuration file
+
+The main configuration file uses the TOML format. The default file name is `.pyodk_config.toml`, and the default location is the user home directory. The file name and location can be customised by setting the environment variable `PYODK_CONFIG_FILE` to some other file path. The expected file structure is as follows:
+
+```
+[central]
```

Comment:
Should we perhaps start right away with a list? It might be simpler if we know we want to support multiple server configs to start there immediately. I'm thinking something like 

```
[servers]
[servers.myprodserver]
base_url = "https://www.example.com"
username = "my_user"
password = "my_password"
default_project_id = 123

[servers.mytestserver]
base_url = "https://www.example.com"
username = "my_user"
password = "my_password"
default_project_id = 123
```

For now we could default to the first one so the actual implementation doesn't have to support multiple ones immediately.

Reply:
Please see other comment about configs